resources/resources/software/aws/toolkits/resources/localized_messages.properties

```diff
@@ -643,6 +643,7 @@ s3.upload.directory.impossible=Cannot upload ''{0}'' because it is a directory
 s3.upload.object.action=Upload...
 s3.upload.object.failed=Failed to upload object {0}
 s3.upload.object.progress=Uploading ''{0}''
+s3.version.history.view=View version history
```

Comment:
I think "Load" would be a better verb here

Reply:
Agree :)